kinetic/distribution.yaml

```diff
@@ -10259,6 +10259,16 @@ repositories:
       url: https://github.com/PX4/px4_msgs-release.git
       version: 1.0.0-3
     status: developed
+  py_script-release:
+    release:
+      packages:
+      - humanoid_shp
+      - humanoid_shp_moveit
+      - py_script
```

Comment:
This is much too generic a name.  This will be used as the package name, so just imagine a user trying to install `ros-kinetic-py-script`; what does that do?  The name of the package should reflect what it is scripts for.